packages/datadog-instrumentations/src/jest.js

```diff
@@ -849,8 +849,6 @@ function cliWrapper (cli, jestVersion) {
     return result
   })
 
-  cli.runCLI = wrapped.runCLI
```

Comment:
I can't understand why this was here previously. Do you know?

Reply:
No, I think it was just a mistake. It didn't change anything anyway.